src/components/ErrorBoundary.jsx

```diff
@@ -27,8 +16,6 @@ class ErrorBoundary extends React.Component {
   }
 
   render() {
-    if (this.state.hasError) return <Error />;
```

Comment:
This is removed so that the error boundary still shows the UI in the case of an error.